README.md

```diff
@@ -1,2 +1,18 @@
-# akka-microservices-seed.g8
-Akka Microservices Giter8 template
+# Akka Microservices Giter8 template
+
+Use with:
+
+```
+sbt new akka/akka-microservices-seed.g8
```

Comment:
We will not support sbt for Java, and it would be akward if we required that they installed sbt just for creating the first project. It will be mvn archetype or whatever is the standard for the other build tools.

We could use -scala for our own naming consistency.

There is one difference from the existing -quickstart. They include a full example, while this is a rather empty project for bootstrapping.